stripe/src/main/java/com/stripe/android/util/DateUtils.java

```diff
@@ -129,6 +129,35 @@ static boolean isExpiryDataValid(int expiryMonth, int expiryYear, @NonNull Calen
     }
 
     /**
+     * Creates a string value to be entered into an expiration date text field
+     * without a divider. For instance, (1, 2020) => "0120". It doesn't matter if
+     * the year is two-digit or four. (1, 20) => "0120".
+     *
+     * Note: A four-digit year will be truncated, so (1, 2720) => "0120".
+     *
+     * @param month a month of the year, represented as a number between 1 and 12
+     * @param year a year number, either in two-digit form or four-digit form
+     * @return a length-four string representing the date
+     */
+    public static String createDateStringFromIntegerInput(
+            @IntRange(from = 1, to = 12) int month,
+            @IntRange(from = 0) int year) {
+        String monthString = String.valueOf(month);
+        if (monthString.length() == 1) {
+            monthString = "0" + monthString;
+        }
+
+        String yearString = String.valueOf(year);
+        if (yearString.length() > 2) {
```

Comment:
You mean, should I defend/fail against that? I suppose -- that would look like an error, and it would be easier to defend against if it just didn't enter a value. I would prefer not to throw an error, since that seems a little heavy handed to me.